<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="txtTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtFrameRate">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>140</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frame Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCount">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>140</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineeditFrameRate">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineeditCount">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>290</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START [Enter]</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listwidgetOutput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>780</width>
      <height>260</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新細明體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>141</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear [Del]</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReencode">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>141</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Re-Encode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
